ui/pages/notifications/notification-components/feature-announcement/feature-announcement.tsx

```diff
@@ -103,28 +103,16 @@ export const components: NotificationComponent<FeatureAnnouncementNotification>
     },
     footer: {
       type: 'footer_feature_announcement',
-      Link: ({ notification }) =>
-        notification.data.link ? (
+      ExtensionLink: ({ notification }) =>
+        notification.data.extensionLink ? (
           <NotificationDetailButton
             notification={notification}
             variant={ButtonVariant.Primary}
-            text={notification.data.link.linkText}
-            href={notification.data.link.linkUrl}
+            text={notification.data.extensionLink.extensionLinkText}
+            href={`/${notification.data.extensionLink.extensionLinkRoute}`}
```

Comment:
Ah, so I guess constructing the `chrome-extension://${id}` part is uneccessary?

Reply:
Exactly!